src/polyfills/details/_base.scss

```diff
@@ -6,6 +6,10 @@
 summary {
 	// Make sure summary remains visible
 	display: list-item !important;
+	list: {
```

Comment:
ok now I understand, but I were confuse when I quickly looked for "style:" instruction in SASS documentation. That landed me to look deeper to see what the community did compared to your propose solution.

May be for that one particular case, because it use a "generic" keyword, I think it will add more clarity if you code twice ```list-style"``` instead of using the SASS nested property extension.

Also, using twice the same CSS property seems to be buggy or to hide somewhat a bug. I know this is not the case here, but to avoid that confusion, what do you think of adding a comment prior those two line of  CSS that explain it?